packages/monorepo-tools/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+const findUp = require('find-up');
+
+exports.MONOREPO_ROOT = path.dirname(
```

Comment:
When used outside of this repo this will point to whatever package.json is found starting from 2 folders above the `node_modules` folder this file is, i'm not sure this is what we want.

If we want't to find the `monorepo` root from cwd, the safest way i can think of is to  `findUp.sync('package-lock.json', {cwd: process.cwd()})` and double-check that it contains workspace info.

Reply:
We use MONOREPO_ROOT in 6 places so I thought the easiest way to get past review would be to try and make a drop-in replacement. I don't mind changing them all.

Reply:
I removed this file now.